app/src/main/java/org/sopt/dateroad/presentation/ui/component/tag/DateRoadDdayTag.kt

```diff
@@ -0,0 +1,51 @@
+package org.sopt.dateroad.presentation.ui.component.tag
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.dateroad.ui.theme.DATEROADTheme
+import org.sopt.dateroad.ui.theme.DateRoadTheme
+import org.sopt.dateroad.ui.theme.DeepPurple
+import org.sopt.dateroad.ui.theme.White
+
+@Composable
+fun DateRoadDdayTag(
+    modifier: Modifier = Modifier,
+    text: String,
+    backgroundColor: Color = DeepPurple,
+    contentColor: Color = White
+) {
+    Row(
+        modifier = modifier
+            .clip(RoundedCornerShape(20.dp))
+            .background(backgroundColor)
```

Comment:
```suggestion
            .background(color = backgroundColor)
```

이런 식으로 어떤 값을 받는지 명시해주면 좋을 것 같아요